services/pull/review.go

```diff
@@ -293,6 +310,10 @@ func SubmitReview(ctx context.Context, doer *user_model.User, gitRepo *git.Repos
 	if reviewType != issues_model.ReviewTypeApprove && reviewType != issues_model.ReviewTypeReject {
 		stale = false
 	} else {
+		if issue.IsClosed || pr.HasMerged {
```

Comment:
Just q question, in which case,  `HasMerged=true` doesn't mean `IsClosed=true`?

Reply:
Initially I was confused what you meant here but now I understand. `IsClosed` will always be set if the PR was merged, so it doesn't make sense to include `pr.HasMerged`. Just to make sure I took a [gander](https://github.com/go-gitea/gitea/blob/27861d711b6284ccc774f974d8a5813ca2c488eb/models/issues/pull.go#L502) at `PullRequest.SetMerged()` and this does appear to be the case after reading this. I'll make the necessary changes!

Reply:
Fixed